package.json

```diff
@@ -66,7 +66,7 @@
   "dependencies": {
     "@bbc/gel-foundations": "4.0.1",
     "@bbc/moment-timezone-include": "^1.1.4",
-    "@bbc/psammead-amp-geo": "^1.1.2",
+    "@bbc/psammead-amp-geo": "^1.2.0",
```

Comment:
Are we okay to bump the amp-geo component in this PR? It looks like it will be bumped in this PR https://github.com/bbc/simorgh/pull/6961. Just making sure there won't be any regression bumping this.

Reply:
@HarveyPeachey Good spot. I'll remove it.